extensions/podman/src/extension.ts

```diff
@@ -986,6 +1009,26 @@ export async function createMachine(
   }
   await execPromise(getPodmanCli(), parameters, { logger, env }, token);
 }
+
+export async function checkMachineParameters(items: AuditRequestItems, podmanVersion?: string): Promise<AuditResult> {
+  const result = {
+    records: [],
+  };
+  if (items['podman.factory.machine.user-mode-networking'] && !isUserModeNetworkingSupported(podmanVersion)) {
```

Comment:
this part is a bit confusing, should we do !(isWindows()) before all the stuff below and then do isUserModeNetworkingSupported?